solutions/swb-app/src/dataSets/createDataSetRequestParser.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+import { z } from 'zod';
+
+// eslint-disable-next-line @rushstack/typedef-var
+export const CreateDataSetRequestParser = z.object({
+  name: z.string(),
```

Comment:
do we want strict in here?

Reply:
Good catch. Adding that in with the next PR